OptimizelySDK/Odp/LruCache.cs

```diff
@@ -0,0 +1,128 @@
+﻿/* 
+ * Copyright 2022, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using OptimizelySDK.Logger;
+using System;
+using System.Collections.Specialized;
+
+namespace OptimizelySDK.Odp
+{
+    public class LruCache<T> : ICache<T>
+    {
+        public const int DEFAULT_MAX_SIZE = 10000;
+        public const int DEFAULT_TIMEOUT_SECONDS = 600;
```

Comment:
Use TimeSpan.

Reply:
Can you clarify your thoughts here?  DM me if it's easier.